test/azul_test_case.py

```diff
@@ -166,6 +167,7 @@ def tearDownClass(cls) -> None:
         cls._restore_aws_account()
         cls._restore_catalogs()
         cls._restore_partition_prefix_length()
+        cls._restore_dss_query_prefix()
```

Comment:
Tear down order is the inverse of set up. Please add the patch below as a first commit and then put your addition in the right place.

```diff
Index: test/azul_test_case.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test/azul_test_case.py b/test/azul_test_case.py
--- a/test/azul_test_case.py	(revision da4a172447844e717d202c57cb3c66ed70f64e6d)
+++ b/test/azul_test_case.py	(date 1632950784268)
@@ -161,11 +161,11 @@
 
     @classmethod
     def tearDownClass(cls) -> None:
+        cls._restore_partition_prefix_length()
         cls._restore_aws_region()
         cls._restore_aws_credentials()
-        cls._restore_aws_account()
+        cls._restore_aws_account_id()
         cls._restore_catalogs()
-        cls._restore_partition_prefix_length()
         super().tearDownClass()
 
     def setUp(self) -> None:
@@ -237,7 +237,7 @@
         os.environ['AZUL_AWS_ACCOUNT_ID'] = moto.core.models.ACCOUNT_ID
 
     @classmethod
-    def _restore_aws_account(cls):
+    def _restore_aws_account_id(cls):
         os.environ['AZUL_AWS_ACCOUNT_ID'] = cls._aws_account_id
 
     get_credentials_botocore = None
```